### ISSUE TITLE
Add max length to input box

### ISSUE THREAD

**Issue by BadgerCode** [BadgerCode/emojichat-html#6]
I believe the limit is 255 **bytes**.

**Comment by BadgerCode** [comment]
Also, apply word wrap/break to long words.

**Comment by BadgerCode** [comment]
Results from testing:
```
Default chat box: (126)
123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456

Max output from custom: (126)
123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456

Max emojis: (31 + 0.5)
🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊🦊?

Max two-byte chars: (63)
īīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīīī
```

It seems that 126 bytes is the max in Garry's Mod.
The Fox emoji is 4 bytes.

**Comment by BadgerCode** [comment]
Also maybe put the : after a user's name in the default text colour

**Comment by BadgerCode** [comment]
Fixed in 7ef2e00
